usb/usb_host_msc/src/msc_host.c

```diff
@@ -76,6 +77,11 @@ static const usb_standard_desc_t *next_endpoint_desc(const usb_standard_desc_t *
     return usb_parse_next_descriptor_of_type(desc, len, USB_B_DESCRIPTOR_TYPE_ENDPOINT, (int *)offset);
 }
 
+static inline bool is_in_endpoint(uint8_t endpoint)
```

Comment:
Nitpick: there is a usb_helper macros n usb_types_ch9.h for that i: USB_EP_DESC_GET_EP_DIR()